ChangeLog

```diff
@@ -13,6 +13,8 @@ Default behavior changes
      Applied Sciences).
 
 Security
+   * Fix buffer overread vulnerability in ssl_parse_server_key_exchange
+     function.
```

Comment:
Same remarks as https://github.com/ARMmbed/mbedtls/pull/1440/files/5224a7544c95552553e2e6be0b4a789956a6464e..999a3bce6e19e3300828584648b20d2784bf882f#r174206943 and https://github.com/ARMmbed/mbedtls/pull/1440/files/5224a7544c95552553e2e6be0b4a789956a6464e..999a3bce6e19e3300828584648b20d2784bf882f#r174207812